src/components/HelpPopup.vue

```diff
@@ -68,7 +89,7 @@ const closeModal = () => dialog.value?.close();
             <Transition>
               <img
                 v-if="openedOnce"
-                :alt="`Where to find ${input}`"
+                :alt="t('translation.wheretofind') ` ${translatedInput}`"
```

Comment:
Does this work? Doesn't seem like valid syntax to me.

```suggestion
                :alt="`${t('translation.wheretofind')} ${translatedInput}`"
```